src/pages/resultsView/network/files/mutationMapper.js

```diff
@@ -1114,13 +1114,18 @@ var MutationDetailsTableFormatter = (function()
 		var total = alt + ref;
 		var style = "";
 		var tipStyle = "";
-
-		if (frequency)
-		{
-			style = "mutation_table_allele_freq";
-			text = frequency.toFixed(2);
-			tipStyle = tipClass;
```

Comment:
@coderrsid Rebasing looks good, one more thing you may want to consider. You have added the conditions for the text, that's good. But the `style` and `tipStyle` seems are disappeared in your code. Could you add them back? Other thoughts @adamabeshouse ?

Reply:
yes, @coderrsid what I meant was please "pull" those out, meaning leave those lines in but move them out of the if statement, since they happen no matter what.

Also, your if statement is unnecessarily complex. Look at the conditions and see if you can simplify it. There's a way to just have it be a single `if() { } else { }`

Reply:
Made the changes as @dippindots  @adamabeshouse  suggested and according to @alisman , i have added the condition that if frequency exists, then only check if it's less than 0.01 , else keep it "NA".